CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
 - No security issues fixed!
 
+## [1.3.4] - 2020-12-15
+### Added
+- Upgrade Kotlin to 1.4.21 and Kodein to 7.1.0, apart from other Android dependencies.
+
```

Comment:
One thing regarding this: since we're adding new dependencies and stuff, I might upgrade to 1.4.0 instead of a 1.3.x version, since it may require a bit of tweaking from the users of the lib, it's not just a bugfixing or dependency updates with no effect for the users.
I mean, the injection code will need to be changed to use `DI` instead of `Kodein` and so on, so it's not a change the users won't notice, hence the upgrade to the minor instead of the patch 🤔

Reply:
5f2e43c45d2de522e798fac62786ac27e393d960